lib/jekyll-commonmark.rb

```diff
@@ -12,17 +12,30 @@ def initialize(config)
           @options = [:DEFAULT]
         else
           @options.reject! do |e|
-            unless CommonMarker::Config::Parse.keys.include? e.to_sym
+            unless CommonMarker::Config::Parse.keys.include? e
```

Comment:
We already call `to_sym` on the entirety of what becomes `@options` above, so technically there's still a DoS vector. (This is replicated below.)